src/main/java/Task.java

```diff
@@ -0,0 +1,26 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public void markDone() {
+        this.isDone = true;
+    }
+
+    public void markUndone() {
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " "); // mark done task with X
+    }
+
+    @Override
+    public String toString() {
+        return ("[" + this.getStatusIcon() + "] ");
+    }
```

Comment:
Could the square brackets be added into getStatusIcon() to simplify the code?
```suggestion
    public String getStatusIcon() {
        return (isDone ? "[X]" : "[ ]"); // mark done task with [X]
    }

    @Override
    public String toString() {
        return this.getStatusIcon();
    }
```